pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>adobe</groupId>
     <artifactId>component-generator</artifactId>
-    <version>1.0</version>
+    <version>1.1-SNAPSHOT</version>
```

Comment:
Should we just go with 1.1?

Reply:
Best practice is to keep it on snapshot on a branch and switch to release version for tagged commits.

Reply:
how much effort would it be to set up downloadable releases for this project?

Reply:
Not a whole lot of effort. With the level of commit activity you could basically manage it manually.

Reply:
![image](https://user-images.githubusercontent.com/524972/113449632-1bd05800-93b3-11eb-8140-0cc980c39f26.png)

Get here by clicking the Code <> tab, then the small Tags link